modules/developers-guide/examples/access-hello/main.mo

```diff
@@ -4,7 +4,7 @@ import Error "mo:base/Error";
 import List "mo:base/List";
 import Prim "mo:prim";
 
-actor {
+shared { caller = initializer } actor class() {
```

Comment:
That is deprecated syntax. Now needs to be
```
shared({caller = initializer}) actor class() {
```
Though I would rather suggest
```suggestion
shared(msg) actor class() {
```
(Other uses of `shared` in this file are outdated as well.)

Reply:
You'll need to change the reference to `initializer` to `msg.caller` (or add the line `let initializer = msg.caller` to the actor body)

Reply:
> or add the line let initializer = msg.caller to the actor body

and change the other uses of `caller` into uses of `initializer`

cc @lsgunnlsgunn